services/src/main/resources/migration-paths/migration-paths.xml

```diff
@@ -40,25 +40,25 @@
     </path>
```

Comment:
I intentionally left `Red Hat` word there for targets where other vendors are mentioned.